internal/factory/resource_names.go

```diff
@@ -16,6 +16,8 @@ type ForwarderResourceNames struct {
 	ServiceAccount                   string
 	InternalLogStoreSecret           string
 	ServiceAccountTokenSecret        string
+	ForwarderName                    string
+	Namespace                        string
```

Comment:
Why do we need to carry the Namespace as part of the naming strategy since we already have it as part of the CLF?

Reply:
We need it in CollectorVisitor https://github.com/openshift/cluster-logging-operator/blob/master/internal/collector/vector/visitors.go#L16

Reply:
Can we pass the namespace to visitors.go without too much rework?  the name factory does not have need of NS to name resources

Reply:
done